### PR TITLE
Signing the releases

### PR DESCRIPTION
**Pull request by ehsavoie**
Adding some annotationsto help with ArtifactHub
Signing our releases

### CODE REVIEW

**Comment by jmesnil** [comment]
The action has errors that look suspicious: https://github.com/wildfly/wildfly-charts/actions/runs/9401623931/job/25893837242?pr=138#step:4:78

File: .github/workflows/helm.yaml [hunk 1]
```diff
@@ -16,23 +16,42 @@ jobs:
...
         with:
           version: v3.4.0
 
       - name: Run chart-releaser
-        uses: helm/chart-releaser-action@v1.1.0
-        with:
-          charts_repo_url: http://docs.wildfly.org/wildfly-charts/
+        uses: helm/chart-releaser-action@v1.6.0
+       # with:
+       #   charts_repo_url: http://docs.wildfly.org/wildfly-charts/
```
[the remark this answers is not in the document]
Comment (ehsavoie) [reply .github/workflows/helm.yaml:55]: it will use the gh_pages of the current git repo

**Comment by ehsavoie** [comment]
@jmesnil I've fixed the errors that were due to bats trying to install detik which we aren't using

File: .github/workflows/helm.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (jmesnil) [reply .github/workflows/helm.yaml:55]: what would be the url? https://wildfly.github.io/wildfly-charts or something else?

**Review by jmesnil — APPROVED** [review]

**Pull request merged**